rotkehlchen/globaldb/handler.py

```diff
@@ -368,9 +382,9 @@ def retrieve_assets(userdb: 'DBHandler', filter_query: 'AssetsFilterQuery') -> t
 
         with userdb.conn.read_ctx() as cursor:
             globaldb = GlobalDBHandler()
-            globaldb.conn.execute('PRAGMA wal_checkpoint;')  # needed to have up to date information when attaching  # noqa: E501
+            globaldb._wal_checkpoint(conn=globaldb.conn)
             cursor.execute(
-                f'ATTACH DATABASE "{globaldb.filepath()!s}" AS globaldb KEY "";',
+                f'ATTACH DATABASE "file:{globaldb.filepath()!s}?more=ro" AS globaldb KEY "";',
```

Comment:
why this change? What does it do? This needs a comment in the code explaining it.

Reply:
![imagen](https://github.com/rotki/rotki/assets/5068010/86c87e22-7384-415f-a38e-ef98a05fe197)

Reply:
![imagen](https://github.com/rotki/rotki/assets/5068010/198cbe00-4eec-48cf-b0b5-e4af415a38e4)

Reply:
Is not needed. It was my fault. As stated in the edit I was trying different things and when setting the mode to read only in the connection string of the attachment I didn't see the -whl file generated but since I was working with 3 databases I mixed the one being attached here. This is not needed